Detectors/gconfig/include/SimSetup/GlobalSimParam.h

```diff
@@ -0,0 +1,53 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef DETECTORS_GCONFIG_INCLUDE_SIMSETUP_GLOBALSIMPARAM_H_
+#define DETECTORS_GCONFIG_INCLUDE_SIMSETUP_GLOBALSIMPARAM_H_
+
+#include "SimConfig/ConfigurableParam.h"
+#include "SimConfig/ConfigurableParamHelper.h"
+
+namespace o2
+{
+
+struct GlobalSimParam : public o2::conf::ConfigurableParamHelper<GlobalSimParam> {
+  int mPAIR = 1;
```

Comment:
since these are structs, we don't need the "m" convention. 
Also wondering if "GlobalSim" is the right word. Something like "SimProcessParam" might be slightly better.

Reply:
@sawenzel I don't know whether "SimProcessParam" is the right term either since "Process" as well as "Cut" have a dedicated meaning each wrt GEANT3/4. So could we go with "SimProcessCutParam"? Or, analogously to the "HallSimParam" could we call it "GlobalProcessCutSimParam"? That means the key of this parameter pack would become "GlobalProcessCut".